components/esp_event/include/esp_event_legacy.h

```diff
@@ -226,6 +226,15 @@ typedef esp_err_t (*system_event_cb_t)(void *ctx, system_event_t *event);
  */
 esp_err_t esp_event_loop_init(system_event_cb_t cb, void *ctx) __attribute__ ((deprecated));
 
+/**
+ * @brief Deinitializes the event loop library
+ *
+ * @return 
+ *  - ESP_OK: Success
+ *  - Others: Fail
+ */
+esp_err_t esp_event_loop_deinit();
```

Comment:
One small request, could you please add `__attribute__((deprecated))` here?

Reply:
Sure. I will update it.